src/core/components/navigator-stack/router.tsx

```diff
@@ -9,6 +9,16 @@
 import * as NavigatorService from 'hyperview/src/services/navigator';
 import * as Types from './types';
 import { StackNavigationState, StackRouter } from '@react-navigation/native';
+import { LOCAL_NAME } from 'hyperview/src/types';
+
+type Route = {
```

Comment:
do we not have a similar type defined already somewhere else? should this belong in `types.ts`?

Reply:
Marked as TODO for now